client/src/main/java/io/pravega/client/stream/EventStreamWriter.java

```diff
@@ -93,7 +93,7 @@
      * 
      * @param transactionId The result retained from calling {@link Transaction#getTxnId()}
      * @return Transaction object with given UUID
-     * @deprecated Use {@link ClientFactory#createTransactionalEventWriter(String, Serializer, EventWriterConfig)} instead
+     * @deprecated Use {@link EventStreamClientFactory#createTransactionalEventWriter(String, Serializer, EventWriterConfig)} instead
```

Comment:
Writer id is part of this  API. This needs to be added, right?
Similar comment related to previous api.